code/controllers/configuration.dm

```diff
@@ -69,11 +69,11 @@ var/list/gamemode_cache = list()
 	var/static/allow_ai_shells = FALSE			// allow AIs to enter and leave special borg shells at will, and for those shells to be buildable.
 	var/static/give_free_ai_shell = FALSE		// allows a specific spawner object to instantiate a premade AI Shell
 	var/static/hostedby = null
-	

```

Comment:
Only changes to this file are whitespace.

Reply:
welcome to byond